src/main/webapp/css/main.css

```diff
@@ -14,8 +14,20 @@
  * limitations under the License.
  */
 
+/*
+  Setting our title theme */
+title {
+    font-family: Futura-PT;
+    font-stretch: condensed;
+    font-kerning: normal;
+    text-align: right;
+    /* background-image: url("https://mir-s3-cdn-cf.behance.net/project_modules/1400/a4b55761858317.5a7c23244e1b0.jpg") ;
+    background-attachment: scroll ;
+    */
+}
 /* Remove default styling from the body tag. */
 body {
+    background-color: cornsilk;
```

Comment:
In order to ensure that the color is interpreted consistently across different browsers, OSes, etc. it is preferable to refer to colors using RGB or RGBA format (often expressed in hex like `#RRGGBB` or using `rgb(red_val, green_val, blue_val)` or `rgba(red_val, green_val, blue_val, alpha_val)`).

For example, the color black can be expressed as `#000000` (which can be simplified to `#000`) or as `rgb(0, 0, 0)` or as `rgba(0, 0, 0, 1)`. By contrast, white can be expressed as `#ffffff` (which can be simplified to `#fff`) or as `rgb(255, 255, 255)` or `rgba(255, 255, 255, 1)`. The highest intensity red can be expressed as `#ff0000` (or simply `#f00`) or `rgb(255, 0, 0)`.